engine/include/engine/resources/ResourcesController.hpp

```diff
@@ -45,7 +45,7 @@ namespace engine::resources {
         Texture *texture(const std::string &name,
                          const std::filesystem::path &path = "",
                          TextureType texture_type          = TextureType::Regular,
-                         bool flip_uvs                     = false);
+                         bool flip_uvs                     = true);
```

Comment:
[CRITICAL] Da li je ovo zaista neophodno kao podrazumevano ponašanje? Trenutno pretpostavljamo da su UV koordinate pravilno orijentisane i radimo flip samo u izuzetnim slučajevima.

Ako Vam je potrebno da UV koordinate budu obrnute za određene modele, možda bi bilo bolje omogućiti kontrolu nad tim putem Config fajla, umesto da to bude globalno zadato ponašanje.